Dockerfile

```
@@ -280,9 +280,10 @@ RUN \
     # Create a git config file in a location accessible for $HOME-less users
     # Ref: https://git-scm.com/docs/git-config#FILES
     mkdir -vp "${XDG_CONFIG_HOME}/git" && touch "${XDG_CONFIG_HOME}/git/config"; \
-    # Ensure the XDG directories have permissions for non-root users
+    # Ensure the XDG and Corepack directories have permissions for non-root users
```

"Corepack directories" feels misleading since this code only changes the permissions of a single corepack file... But wording it correctly feels difficult... So I'd probably just let the code speak for itself as far as which directories are affected

```suggestion
    # Ensure non-root users have necessary permissions
```